src/network/WiFiHandler.cpp

```diff
@@ -0,0 +1,229 @@
+/*
+    SlimeVR Code is placed under the MIT license
+    Copyright (c) 2022 TheDevMinerTV
```

Comment:
Exclude personal copyright identifier, it sets a bad precedent where all files will need to include this. Your copyright is present regardless of whether its written in the file. You never signed a contributor license agreement.

TBH I wouldn't include the MIT header either, but thats up to @Eirenliel